src/plugins/workspace/server/saved_objects/workspace_id_consumer_wrapper.ts

```diff
@@ -37,6 +41,24 @@ export class WorkspaceIdConsumerWrapper {
       ...(finalWorkspaces.length ? { workspaces: finalWorkspaces } : {}),
     };
   }
+  private isDataSourceType(type: SavedObjectsFindOptions['type']): boolean {
+    if (Array.isArray(type)) {
+      return type.every((item) => item === DATA_SOURCE_SAVED_OBJECT_TYPE);
+    }
+
+    return type === DATA_SOURCE_SAVED_OBJECT_TYPE;
+  }
+  private isConfigType(type: SavedObjectsFindOptions['type']): boolean {
```

Comment:
Type here should be `SavedObjectsBulkCreateObject['type']` ?

Reply:
Sure, changed to `SavedObject['type']`.